genienlp/validate.py

```diff
@@ -61,12 +62,19 @@ def generate_with_model(model, data_iterator, numericalizer, task,args,
     answers = []
     contexts = []
 
-    for batch in data_iterator:
+    for batch in progress_bar(data_iterator, desc='Generating'):
```

Comment:
Will this affect `genienlp server` and cause us to spit out a line for every prediction?

Reply:
Yes, in stderr. Should I disable it for server mode?

Reply:
Yes please. It will generate a lot of noise otherwise, because it starts and stops a new progress bar for every sentence/minibatch.